README.md

```diff
@@ -191,8 +197,8 @@ env $(cat /etc/default/influxdb | xargs) influxd -config /etc/influxdb/influxdb.
 On a host in your NuoDB domain, install Grafana and configure it to use the NuoDB-Insights dashboards.
 
 ```
-wget https://dl.grafana.com/oss/release/grafana-7.5.4-1.x86_64.rpm
-sudo yum install grafana-7.5.4-1.x86_64.rpm
+wget https://dl.grafana.com/enterprise/release/grafana-enterprise_9.5.6_amd64.deb
+sudo yum install -y https://dl.grafana.com/enterprise/release/grafana-enterprise-9.5.6-1.x86_64.rpm
```

Comment:
I notice that we change from "oss" to "enterprise". Does this mean anything in terms of licensing?